### DISCUSSION TITLE
How to set relative BUNDLE_PATH that works both locally and with GitHub Actions?

### DISCUSSION THREAD

**Discussion by eliduke** [ruby/setup-ruby#576] [Q&A] [answered]
I am configuring a super basic ruby workflow as such:

```
- name: Set up Ruby
  uses: ruby/setup-ruby@v1
  with:
    bundler-cache: true
```

That's great, but then it automatically creates a `.bundle/config` file that sets the BUNDLE_PATH to something *incredibly specific* to GitHub Actions. That config file is now included in version control and completely throws off my local development:

```
---
BUNDLE_PATH: "/home/runner/work/repo-name/repo-name/vendor/bundle"
BUNDLE_DEPLOYMENT: "true"
```

Is there a way to set a *relative* bundle path? I tried editing that file and changing the path to `./vendor/bundle`, and the workflow succeeds, but then it resets back to that same specific bundle path. OR is there a way to set a local path and then a different one for GitHub Actions? Or, maybe I need to delete the config completely, add `.bundle/config` to my `.gitignore`, turn bundler cache back on, and then run it all again?

Thanks for reading my rambling! Does any of this make sense? Thoughts? Suggestions?

**Comment by dentarg** [comment, marked as answer]
The general advice from the Bundler project is to not commit `.bundle/config` – if that solves it for you I think you should go for that

**Comment by eliduke** [comment]
Yeah, I think my problem is that I have a ruby workflow that might create some new files and then auto commits anything new. On the first run, it committed the bundle config. Ooops!

I got it all reset and sorted and it’s great now.

thanks!
